templates/d8/plugin/views/sort-plugin.twig

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Drupal\{{ machine_name }}\Plugin\views\sort;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\views\Plugin\views\sort\SortPluginBase;
+
+/**
+ * {{ plugin_label }} sort plugin.
+ *
+ * @ViewsSort("{{ plugin_id }}")
+ */
+class {{ class }} extends SortPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $usesRowPlugin = TRUE;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $usesRowClass = TRUE;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function defineOptions() {
+    $options = parent::defineOptions();
+    return $options;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildOptionsForm(&$form, FormStateInterface $form_state) {
```

Comment:
This method is an exact copy of `SortPluginBase::buildOptionsForm()`, I dont think we need to add it to the generated plugin.